packages/cli/src/codemods/versions/33.0.0/manifest.ts

```diff
@@ -5,17 +5,77 @@ import transformGridApiMethodsV33_0 from '../../transforms/transform-grid-api-me
 import transformSparklinesOptionsV33_0 from '../../transforms/transform-sparklines-options-v33-0/manifest.ts';
 
 import transformModulesToPackagesV33 from '../../transforms/transform-modules-to-packages-v33/manifest.ts';
+import { select } from '@inquirer/prompts';
 
 const transforms: Array<TransformManifest> = [
   transformGridApiMethodsV33_0,
   transformSparklinesOptionsV33_0,
   transformModulesToPackagesV33,
 ];
 
-const manifest: VersionManifest = {
+type Choices33 = {
+  usingCharts: boolean;
+  chartType: string;
+};
+
+const manifest: VersionManifest<Choices33> = {
```

Comment:
Idea is to have each version able to ask the user questions for input. These can then be used to set global variables that can be used within the transforms themselves to tweak behaviour.